source/playbooks/SC/ssmdocs/SC_Redshift.4.ts

```diff
@@ -95,7 +109,7 @@ class EnableRedshiftClusterAuditLoggingDocument extends ControlRunbookDocument {
   protected getRemediationParams(): { [_: string]: any } {
     const params = super.getRemediationParams();
 
-    params.MinRetentionPeriod = StringVariable.of('ExtractConfigRuleParameters.RetentionPeriodSerialized');
+    params.MinRetentionPeriod = StringVariable.of('ExtractConfigRuleParameters.MinRetentionPeriod');
```

Comment:
Are we sure this is right? looking at AFSBP_Redshift.4.yaml, the remediation takes parameters `ClusterIdentifier` and `AutomationAssumeRole`, which are handled automatically by the parent class. The other parameter to add here would be `BucketName` with the value `{{CheckIfSSMParameterWithS3BucketNameIsAvailable.BucketName}}`.

Reply:
Reviewing how I worked out this issue from yesterday, there may be a chance that this is incorrect. The issue could've been just that RetentionPeriodSerialized was missing from the parse input, a change that I made after changing the parameter names. Something odd is that this did pass so I'm wondering if this value is even used at all.